pkg/syncer/cnsoperator/controller/cnsnodevmattachment/cnsnodevmattachment_controller.go

```diff
@@ -184,8 +194,8 @@ func (r *ReconcileCnsNodeVMAttachment) Reconcile(ctx context.Context, request re
 
 	vcdcMap, err := getVCDatacentersFromConfig(r.configInfo.Cfg)
 	if err != nil {
-		msg := fmt.Sprintf("failed to find datacenter moref from config for CnsNodeVmAttachment request with name: %q on namespace: %q. Err: %+v",
-			request.Name, request.Namespace, err)
+		msg := fmt.Sprintf("failed to find datacenter moref from config for CnsNodeVmAttachment "+
+			"request with name: %q on namespace: %q. Err: %+v", request.Name, request.Namespace, err)
```

Comment:
Did you miss a full stop at line# 212?

// Get node VM by nodeUUID

Reply:
Good catch. Fixed.